.github/workflows/main.yml

```diff
@@ -66,14 +66,10 @@ jobs:
 
     if: github.ref == 'refs/heads/main'
 
-    outputs:
-      tag: ${{ steps.tag.outputs.new_version }}
-
     steps:
       - uses: actions/checkout@v3
-      - name: Bump version and push tag
-        id: tag
-        uses: mathieudutour/github-tag-action@d745f2e74aaf1ee82e747b181f7a0967978abee0
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          create_annotated_tag: true
+      - name: Tag new release
+        run: |
+          # ignore failures here to avoid merges into main without version
+          # bumps failing this job.
+          git tag "v$(cat version)" || exit
```

Comment:
I think you'll also need `git push --tags` too?

Or else the tag will never be applied to the repo?

Reply:
The action seems to create then push: https://github.com/mathieudutour/github-tag-action/blob/master/src/github.ts#L68

Reply:
D'oh, yep!